tap_github/client.py

```diff
@@ -259,23 +259,29 @@ def get_all_repos(self, organizations: list):
 
         for org_path in organizations:
             org = org_path.split('/')[0]
-            for response in self.authed_get_all_pages(
-                'get_all_repos',
-                '{}/orgs/{}/repos?sort=created&direction=desc'.format(self.base_url, org)
-            ):
-                org_repos = response.json()
-                LOGGER.info("Collecting repos for organization: %s", org)
-
-                for repo in org_repos:
-                    repo_full_name = repo.get('full_name')
-                    LOGGER.info("Verifying access of repository: %s", repo_full_name)
-
-                    self.verify_repo_access(
-                        '{}/repos/{}/commits'.format(self.base_url,repo_full_name),
-                        repo
-                    )
-
-                    repos.append(repo_full_name)
+            try:
+                for response in self.authed_get_all_pages(
+                    'get_all_repos',
+                    '{}/orgs/{}/repos?sort=created&direction=desc'.format(self.base_url, org),
+                    should_skip_404 = False
+                ):
+                    org_repos = response.json()
+                    LOGGER.info("Collecting repos for organization: %s", org)
```

Comment:
It should be before API calls, else update Collecting to Collected.

Reply:
Updated logger message.